### PR TITLE
Fix file names following AirBnb naming conventions (2) 

### PR DESCRIPTION
**Pull request by neonima**
This PR intends to correct the project file names following the AirBnb naming convention: https://github.com/airbnb/javascript#naming-conventions

It also adds explicit member rule in the TS Lint (`member-access:true`) as changed in (https://github.com/codelittinc/ay-properties-api/pull/19)

### CODE REVIEW

File: tests/integration/models/base.model.test.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { User as BaseModel } from '@models/user.model';
+import { User as BaseModel } from '../../../src/models/UserModel';
```
Comment (alesmit) [inline tests/integration/models/base.model.test.ts]: use shorthand import `@models/userModel`

File: tests/unit/controllers/user.controller.test.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import { getApp } from '@server';
-import { User } from '../../../src/models/user.model';
+import { User } from '../../../src/models/UserModel';
```
Comment (alesmit) [inline tests/unit/controllers/user.controller.test.ts]: please use shorthand import here as well

File: tests/integration/models/user.model.test.ts [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import { User } from '@models/user.model';
+import { User } from '../../../src/models/UserModel';
```
Comment (alesmit) [inline tests/integration/models/user.model.test.ts]: please use shorthand import here as well

File: src/db/seeds/index.ts [hunk 4]
```diff
@@ -1,4 +1,4 @@
-import UserSeed from './users.seed';
-import PropertySeed from './properties.seed';
+import UserSeed from './usersSeed';
+import PropertySeed from './propertiesSeed';
```
Comment (kaiomagalhaes) [inline src/db/seeds/index.ts]: shouldn't it be propertySeed ?

**Pull request merged**